src/XCMRelayer.sol

```diff
@@ -209,4 +268,120 @@ contract AxelarXCMRelayer is Auth {
     function _parachainId() internal pure returns (bytes memory) {
         return abi.encodePacked(uint8(0), CENTRIFUGE_PARACHAIN_ID);
     }
+
+    // --- Utilities ---
+    function _centrifugeCall(
+        bytes memory payloadLength,
+        string memory sourceChain,
+        string memory sourceAddress,
+        bytes memory message
+    ) internal pure returns (bytes memory) {
+        return abi.encodePacked(
+            hex"7305",
+            payloadLength,
+            bytes32(bytes(sourceChain).length),
+            bytes(sourceChain),
+            bytes32(bytes(sourceAddress).length),
+            bytes(sourceAddress),
+            message
+        );
+    }
+
+    // Obtain the Scale-encoded length of a given message. Each Liquidity Pools Message is fixed-sized and
+    // have thus a fixed scale-encoded length associated to which message variant (aka Call).
+    //
+    // TODO(Nuno): verify this is still correct and that we support all new incoming messages
+    function messageLengthScaleEncoded(bytes memory _msg) internal pure returns (bytes memory) {
```

Comment:
We probably don't need this in the end but only need to specify the scale-encoded length of the whole payload. will remove once we figured this whole thing out